### PR TITLE
Update mock expectation post #818

### PR DESCRIPTION
**Pull request by michaelklishin**

### CODE REVIEW

File: tests/unit/channel_tests.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-"""
+gi"""
```
Comment (lukebakken) [inline tests/unit/channel_tests.py]: What does this change do?

Comment (michaelklishin) [inline tests/unit/channel_tests.py]: Err, looks like an Emacs artefact. Will fix.

**Comment by michaelklishin** [comment]
@lukebakken thanks for spotting the typo, ready for another round.

File: tests/unit/channel_tests.py [hunk 2]
```diff
@@ -522,7 +522,9 @@ def test_close_basic_cancel_called(self):
         self.obj._consumers['abc'] = None
         with mock.patch.object(self.obj, 'basic_cancel') as basic_cancel:
             self.obj.close()
-            basic_cancel.assert_called_once_with(consumer_tag='abc', nowait=True)
+            # this is actually not necessary but Pika currently cancels
+            # every consumer before closing the channel
+            basic_cancel.assert_called_once_with(consumer_tag='abc', nowait = True)
```
Comment (lukebakken) [inline tests/unit/channel_tests.py]: This is super nit-picky, but typical Python style for parameter default assignment is to not use spaces.

**Pull request merged**